page-objects/modals/editSpaceNameModal.ts

```diff
@@ -0,0 +1,32 @@
+import { Locator, Page } from "@playwright/test";
+
+export class EditSpaceNameModal {
+  private readonly page: Page;
+  private readonly modalContainer: Locator;
+  private readonly spaceNameInput: Locator;
+  private readonly saveButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.modalContainer = this.page.getByTestId("modal__dialog");
+    this.spaceNameInput = this.modalContainer.getByTestId("create-project-modal__form-input");
+    this.saveButton = this.modalContainer.getByTestId("create-project-modal__next-step-create-new-space");
+  }
+
+  async typeSpaceName(spaceName: string) {
+    await this.modalContainer.waitFor();
+    await this.spaceNameInput.waitFor();
+    await this.spaceNameInput.clear();
+    await this.spaceNameInput.fill(spaceName);
+  }
+
+  async clickOnSaveButton() {
+    await this.modalContainer.waitFor();
+    await this.saveButton.waitFor();
+    await this.saveButton.click();
```

Comment:
Do ustalenia czy wrzucamy wrzędzie waity, afaik to playwright używając metody .click() ma wbudowane czekanie aż element się pojawi

Reply:
Słuszna uwaga - usunę te waity i sprawdzę, czy ma to jakiś wpływ na testy (nie powinno mieć)